### Calculating Probabilities

#### Salmon Fish Example

In [16]:
import numpy as np
import matplotlib.pyplot as plt
from scipy import stats

In [17]:
x = 75
mean = 60
std_dev = 40
samp_size = 10
standard_error = std_dev / (samp_size**.5)

stats.norm.cdf(x, mean, standard_error)



0.8821600432854813

The probability that the average weight of our sample of 10 fish will be less than or equal to 75 lbs is 88.2%.

#### Cod Fish Example

In [18]:
x = 30
mean = 36
std_dev = 20
samp_size = 25


standard_error = std_dev / (samp_size**.5)
print(standard_error)
cod_cdf = stats.norm.cdf(x, mean, standard_error)
print(cod_cdf)

4.0
0.06680720126885807


The probability of 25 fish having an average weight of 30 or less is 10%. Based on this, it’s probably a good idea to put less fish in the crate or to get a sturdier crate.